scss/_patterns_breadcrumbs.scss

```diff
@@ -7,15 +7,16 @@
     margin: 0;
     padding: 0;
     width: 100%;
+  }
 
-    &__items {
-      margin-left: 0;
-      padding-left: 0;
-    }
+  .p-breadcrumbs__items {
+    margin-bottom: $spv--large;
+    margin-left: 0;
+    padding-left: 0;
   }
 
   .p-breadcrumbs__item {
-    @extend %default-text;
+    @extend %small-caps-text;
 
     display: inline-block;
     margin-bottom: $spv-nudge-compensation;
```

Comment:
I think this should be removed, this is some old margin, and we want the margin from small caps text.